src/ui/wxWidgets/PasswordPolicyDlg.cpp

```diff
@@ -297,7 +298,7 @@ void PasswordPolicyDlg::CreateControls()
 
   m_pwpSymCtrl = new wxCheckBox( itemDialog1, ID_CHECKBOX6, _("Use symbols"), wxDefaultPosition, wxDefaultSize, 0 );
   m_pwpSymCtrl->SetValue(false);
-  m_pwpSymCtrl->Bind(wxEVT_MOTION, [&](wxMouseEvent& event) { m_pwpSymCtrl->SetToolTip(_("i.e., ., %, $, etc.")); });
+  m_pwpSymCtrl->Bind(wxEVT_MOTION, [&](wxMouseEvent& /*event*/) { m_pwpSymCtrl->SetToolTip(_("i.e., ., %, $, etc.")); });
```

Comment:
What about WXUNUSED as it is used already in some other cases?

Reply:
`WXUNUSED` macro comment them too), but, for uniformity, changed commented names to macro (only in wx-related code).